app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str | None:
+    cmd = command.split()
+    if cmd[0] != "mv":
+        return f"{cmd[0]} not a move command"
```

Comment:
Thank you for the review! As for me, this function looks like a "under the hood function", but which works with the user's request, and it would be nice not to just not fulfill the request, but to explain why. Agree that these solutions do not cover all possible input errors. And on 7th line, I did not want to make this decision more complicated and write an additional custom exception. "Simple is better than complex", you know ;)

Reply:
Okay, seems like a valid reasoning. What about my other comment? Do you want to change anything or do I just approve?